compiler/src/iree/compiler/Codegen/Common/Passes.h

```diff
@@ -60,7 +60,8 @@ createConvertToDestinationPassingStylePass(
     bool useWARForCooperativeMatrixCodegen);
 
 std::unique_ptr<InterfacePass<FunctionOpInterface>>
-createDecomposePackUnPackOpsPass(bool tileOuterToOne);
+createDecomposePackUnPackOpsPass(bool tileOuterToOne,
+                                 bool useOnlyReshapes = false);
```

Comment:
I don't like the style that one has default values but the other does not. Perhaps we should just retire this wrapper and use the option path, or remove the default values from `useOnlyReshapes`.

Reply:
I realized that we actually need an optional control function on the decomposition as well, so I think it is better to leave the wrapper now. I can add a default value for `tileOuterToOne`, WDYT?

Reply:
Actually I decided to put the control function as an option, since it is optional. The command line option for it will exist but not be usable, but I think it was cleaner this way.

Reply:
>  I can add a default value for tileOuterToOne, WDYT?

It won't work because it is confusing to the default method. There is already a tablegen-ed definition, and adding a default value for the variable leads to a confusion. You'll likely hit a compilation error. That's why I suggested to remove the default value from `useOnlyReshapes`,